pyspecProcScripts/phasing.py

```diff
@@ -221,7 +253,7 @@ def filter_ranges(B, A):
         return [np.array(b) for b in B if
```

Comment:
I really don't fully understand what this is doing/why it is better than just using contiguous once to decide the range? notes in notebook didn't help